src/wr_logic_teleop_arm/src/SolenoidController.hpp

```diff
@@ -0,0 +1,30 @@
+// Header guard
+#ifndef SOLENOID_CONTROLLER_HPP
+#define SOLENOID_CONTROLLER_HPP
+
+#include "ros/subscriber.h"
+#include "std_msgs/Bool.h"
+#include <fstream>
+
+class SolenoidController {
+public:
+    explicit SolenoidController(ros::NodeHandle&);
+
+    void extendSolenoid(const std_msgs::Bool::ConstPtr& msg);
```

Comment:
This could (and should) probably be private.

Reply:
Fixed in 8edc4f19.